### ISSUE TITLE
CSS incorrectly blocked on fandom.com

### ISSUE THREAD

**Issue by FWDekker** [brave/brave-browser#4480]
## Description 
CSS is blocked by Shields on the wikis hosted on [fandom.com](fandom.com). Shields detects four 3rd-party trackers, one of which hosts the CSS:
```
https://slot1-images.wikia.nocookie.net/__am/1558034982680/sasses/background-dynamic%3Dfalse%26background-image%3Dhttps%253A%252F%252Fimages.wikia.nocookie.net%252F__cb57%252Ffallout%252Fimages%252F5%252F50%252FWiki-background%26background-image-height%3D714%26background-image-width%3D556%26color-body%3D%2523c9cdac%26color-body-middle%3D%2523c9cdac%26color-buttons%3D%2523006722%26color-community-header%3D%2523006722%26color-header%3D%2523044d22%26color-links%3D%2523002bb8%26color-page%3D%2523ffffff%26oasisTypography%3D1%26page-opacity%3D100%26widthType%3D0/skins/oasis/css/oasis.scss,extensions/wikia/DesignSystem/styles/design-system.scss,extensions/wikia/CommunityHeader/styles/index.scss,extensions/wikia/PageHeader/styles/index.scss,extensions/wikia/Recirculation/styles/recirculation.scss,extensions/wikia/EmbeddableDiscussions/styles/EmbeddableDiscussions.scss,extensions/wikia/PortableInfobox/styles/PortableInfobox.scss,extensions/wikia/CommunityPage/styles/firstedit/FirstEditModal.scss,extensions/wikia/GlobalShortcuts/styles/GlobalShortcutsHelp.scss,extensions/wikia/GlobalShortcuts/styles/GlobalShortcutsSearch.scss,extensions/wikia/MyHome/oasis.scss,extensions/wikia/AdEngine3/dist/ads.scss,extensions/wikia/Qualaroo/css/Qualaroo.scss
```

## Steps to Reproduce
1. Ensure that Shields is enabled.
2. Go to any wiki hosted on Fandom such as [Fallout Wiki](fallout.fandom.com), [Pokémon Wiki](pokemon.fandom.com), or [Witcher Wiki](witcher.fandom.com).

## Actual result
![Actual result](https://user-images.githubusercontent.com/13442533/57919912-1b4f7680-789a-11e9-93b2-915da4a06a07.png)

## Expected result
![Expected result](https://user-images.githubusercontent.com/13442533/57920616-c01e8380-789b-11e9-8b4d-9b2657a0777d.png)

## Reproduces how often
Easily reproduced

## Brave version (brave://version info)
```
Brave    0.64.76 Chromium: 74.0.3729.157 (Official Build) (64-bit)
Revision 7b16107ab85c5364cdcd0b2dea2539a1f2dc327a-refs/branch-heads/3729@{#998}
OS       Windows 10 OS Build 17763.503
```

## Version/channel information
- Does it reproduce on brave-browser dev/beta builds?
Did not test.

## Other additional information
- Does the issue resolve itself when disabling Brave Shields?
  Yes.
- Does the issue resolve itself when disabling Brave Rewards?
  No.
- Is the issue reproducible on the latest version of Chrome? 
  No.

**Comment by rebron** [comment]
@FWDekker Can you try this again? We can't reproduce this issue.

**Comment by FWDekker** [comment]
@rebron I can no longer reproduce the issue either. Perhaps it was fixed in 0.64.77? Anyway, issue closed :)
